manual/tracy.tex

```diff
@@ -1692,6 +1692,16 @@ \subsubsection{OpenCL}
 
 Similar to Vulkan and OpenGL, you also need to periodically collect the OpenCL events using the \texttt{TracyCLCollect(ctx)} macro. An excellent place to perform this operation is after a \texttt{clFinish} since this will ensure that any previously queued OpenCL commands will have finished by this point.
 
+\subsubsection{CUDA}
+
+CUDA support is enabled by including the \texttt{public/tracy/TracyCUDA.hpp} header file. Tracing CUDA requires the creation of a Tracy CUDA context using the macro \texttt{TracyCUDAContext()}, which returns an instance of a \texttt{TracyCUDACtx} object. This context must be used when profiling CUDA events. Cleanup is handled using the \texttt{TracyCUDAContextDestroy(ctx)} macro. To assign a custom name to the context, use the \texttt{TracyCUDAContextName(ctx, name, size)} macro.
+
+To begin profiling CUDA events, call the \texttt{TracyCUDAStartProfiling(ctx)} macro. This initiates the profiling of CUDA events, and GPU zones for key CUDA activities will be created automatically.
```

Comment:
~the profiling of CUDA events~
the instrumentation of all CUDA API calls

~and GPU zones for key CUDA activities will be created automatically.~
and for relevant GPU activity (kernel execution, memory transfers, synchronization). This instrumentation is automatic, and no code annotation is needed.

Reply:
I recommend adding a footnote at the end of the sentence to explain why it works this way.
Something like: CUDA does not have an API to retrieve the timestamps associated to events, therefore, the typical GPU instrumentation design of Tracy cannot be realized.